### ISSUE TITLE
Extra Query Parameters are not sent to the /token endpoint

### ISSUE THREAD

**Issue by bgavrilMS** [AzureAD/microsoft-authentication-library-for-js#5405]
### Core Library

MSAL Node (@azure/msal-node)

### Core Library Version

1.14.3

### Wrapper Library

Not Applicable

### Wrapper Library Version

n/a

### Public or Confidential Client?

Confidential

### Description

1. Use the client_credentials sample
2. Try to set query parameters `foo=bar`. It cannot be done.

Actual: token request is made on <env>/token endpoint
Expected: token request is made on <env>/token?foo=bar endpoint

Impact: unable to target AAD test slices

### MSAL Configuration

```javascript
client_creds with extra query parameter configuration is not possible today, the config does not accept it. This prevents testing on AAD test slices.
```

### Relevant Code Snippets

```javascript
n/a
```

### Reproduction Steps

see description

### Expected Behavior

see description

### Identity Provider

Azure AD / MSA

### Browsers Affected (Select all that apply)

None (Server)

### Source

Internal (Microsoft)

**Comment by sameerag** [comment]
@tnorling Is this addressed by #5403?

**Comment by tnorling** [comment]
Nope, this is different
